bitnami/common/templates/_ingress.tpl

```diff
@@ -26,3 +26,17 @@ service:
     {{- end }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Print "true" if the API pathType field is supported
+Usage:
+{{ include "common.ingress.supportsPathType" . }}
+*/}}
+{{- define "common.ingress.supportsPathType" -}}
+{{- $apiVersion := (include "common.capabilities.ingress.apiVersion" .) -}}
+{{- if (eq $apiVersion "extensions/v1beta1") -}}
+{{- print "false" -}}
+{{- else -}}
+{{- print "true" -}}
+{{- end -}}
+{{- end -}}
```

Comment:
Does this address the `pathType` concern? k8s 1.17 would print `true` but `pathType` was added in k8s 1.18

```
❯ k version
Client Version: version.Info{Major:"1", Minor:"17", GitVersion:"v1.17.9", GitCommit:"4fb7ed12476d57b8437ada90b4f93b17ffaeed99", GitTreeState:"clean", BuildDate:"2020-07-15T16:18:16Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"17+", GitVersion:"v1.17.14-gke.1200", GitCommit:"7c407f5cc8632f9af5a2657f220963aa7f1c46e7", GitTreeState:"clean", BuildDate:"2020-12-01T09:20:59Z", GoVersion:"go1.13.15b4", Compiler:"gc", Platform:"linux/amd64"}


❯ k explain ingress.spec.rules.http.paths.pathType
error: field "pathType" does not exist
```

Reply:
pathType was added in `networking/v1beta1` if I'm not mistaken. It k8s 1.17 it would use extensions/v1beta1

Reply:
```
❯ k explain ingress.spec.rules.http.paths --api-version='networking.k8s.io/v1beta1'
KIND:     Ingress
VERSION:  networking.k8s.io/v1beta1

RESOURCE: paths <[]Object>

DESCRIPTION:
     A collection of paths that map requests to backends.

     HTTPIngressPath associates a path regex with a backend. Incoming urls
     matching the path are forwarded to the backend.

FIELDS:
   backend	<Object> -required-
     Backend defines the referenced service endpoint to which the traffic will
     be forwarded to.

   path	<string>
     Path is an extended POSIX regex as defined by IEEE Std 1003.1, (i.e this
     follows the egrep/unix syntax, not the perl syntax) matched against the
     path of an incoming request. Currently it can contain characters disallowed
     from the conventional "path" part of a URL as defined by RFC 3986. Paths
     must begin with a '/'. If unspecified, the path defaults to a catch all
     sending traffic to the backend.
     ```

Reply:
If I'm not missing anything, 1.17 would use `networking/v1beta1`:

```
{{- if .Values.ingress.apiVersion -}}
{{- .Values.ingress.apiVersion -}}
{{- else if semverCompare "<1.14-0" .Capabilities.KubeVersion.Version -}}
{{- print "extensions/v1beta1" -}}
{{- else if semverCompare "<1.19-0" .Capabilities.KubeVersion.Version -}}
{{- print "networking.k8s.io/v1beta1" -}}
{{- else -}}
{{- print "networking.k8s.io/v1" -}}
{{- end -}}
```

Reply:
Added extra check